input_parser.star

```diff
@@ -91,7 +91,8 @@ DEFAULT_PORTS = {
     "zkevm_pprof_port": 6060,
     "zkevm_rpc_http_port": 8123,
     "zkevm_rpc_ws_port": 8133,
-    "zkevm_cdk_node_port": 5576,
+    "cdk_node_rpc_port": 5576,
```

Comment:
should we name it aggkit node?

Reply:
Fixed https://github.com/0xPolygon/kurtosis-cdk/pull/609/commits/612d5fa5be06b08801984fb78d06e2ae77145305